kubernetes-kotlin/src/main/kotlin/myproject/Support.kt

```diff
@@ -0,0 +1,55 @@
+package myproject
+
+import com.pulumi.kubernetes.apps_v1.Deployment
+import com.pulumi.kubernetes.apps_v1.DeploymentArgs
+import com.pulumi.kubernetes.apps_v1.inputs.DeploymentSpecArgs
```

Comment:
I believe more recent versions of com.pulumi:kubernetes upgraded to the following style of imports:

```
import com.pulumi.kubernetes.apps.v1.Deployment
```

Reply:
What version of the com.pulumi:kubernetes artifact is the new package available on ? 
I don't see it in com.pulumi:kubernetes:(,3.20]